jupyterhub/apihandlers/groups.py

```diff
@@ -75,8 +104,37 @@ class GroupAPIHandler(_GroupAPIHandler):
 
     @admin_only
     def get(self, name):
+        offset = self.get_argument("offset", None)
+        limit = self.get_argument("limit", self.settings['app'].api_page_default_limit)
+        max_limit = self.settings['app'].api_page_max_limit
+
         group = self.find_group(name)
-        self.write(json.dumps(self.group_model(group)))
+
+        if offset is not None:
+            try:
+                offset = int(offset)
+            except Exception as e:
+                raise web.HTTPError(
+                    400, "Invalid argument type, offset must be an integer"
+                )
+
+            group.users = group.users[offset:]
```

Comment:
Editing this list attribute is actually changing group membership, which we don't want to do. The magic of ORMs!

Reply:
Good point, will revert this one back for now 👍 Wasn't very happy with the after-the-fact slicing either. You think we should add the number of users int in or save that for later?

Reply:
Yeah, since it's both more complex and less used than the lists themselves, I think we can save it for later.